src/primo/browzine-primo-adapter.js

```diff
@@ -761,6 +761,15 @@ browzine.primo = (function() {
       // 3i Normal Integration: ✓
       // Primo Studio Integration: ✗
       scope = $scope.$parent.$ctrl;
+    } else if ($scope && $scope.$parent && $scope.$parent.$ctrl.parentCtrl && $scope.$parent.$ctrl.parentCtrl.$element) {
```

Comment:
Complete walking the line by adding a `$scope.$parent.$ctrl` check to make sure it exists:
$scope && $scope.$parent **&& $scope.$parent.$ctrl** && $scope.$parent.$ctrl.parentCtrl && $scope.$parent.$ctrl.parentCtrl.$element

Reply:
Whoops, yes, thanks for catching that Tian!